### PR TITLE
I have changed insertRestaurant route to reflect data to be inserted.

### PR DESCRIPTION
**Pull request by gdeepchadha**
I have also removed useless get method in restaurant route.

### CODE REVIEW

File: routes/insertRestaurant.js [hunk 1]
```diff
@@ -69,4 +39,370 @@ router.get('/', function(req, res, next){
...
+        },
+        {
+            name:'cheese subway sandwich',
+            description:'cheese'
+        }],
+        cuisine:'American'
+    };
+}
+
 module.exports=router;
```
Comment (GITANG1) [inline routes/insertRestaurant.js]: EOL

File: routes/restaurants.js [hunk 2]
```diff
@@ -143,4 +107,4 @@ searchRouter.route('/')
 
 });
 
-module.exports = restaurantRouter;
+module.exports = restaurantRouter;
```
Comment (GITANG1) [inline routes/restaurants.js]: EOL

File: routes/insertRestaurant.js [hunk 3]
```diff
@@ -69,4 +39,370 @@ router.get('/', function(req, res, next){
...
+        state:'Florida',
+        timing:[{
+            open:800,
+            close: 2200
+        }],
+        phone:8503062272,
+        expense_level:2,
+        rating:2,
+        menu:[{
+            name:'veggie Pizza',
```
Comment (GITANG1) [inline routes/insertRestaurant.js:98]: Veggie should be capitalized

File: routes/insertRestaurant.js [hunk 4]
```diff
@@ -23,42 +27,8 @@ router.get('/', function(req, res, next){
...
-             {
-                 name:'Enchiladas',
-                 description:'Stuffed with meat, cheese and sauce. Served with cilantro lime rice, and choice of beans. SAUCES: Roasted Green Chile, Tomatillo Cilantro, Red Enchilada'
-             }]
-        }, function (err, restaurant) {
+        Restaurants.create(jsonArray[0]
+        , function (err, restaurant) {
             if (err) throw err;
             console.log('Restaurant created!');
             console.log(restaurant);
```
Comment (srishtihunjan) [inline routes/insertRestaurant.js]: do we need both the logs? They're at the same location and i'm guessing for the same thing

File: routes/restaurants.js (hunk 2 above)
Comment (GITANG1) [reply routes/restaurants.js]: -remove comments if not contributing to functionality.

**Review by GITANG1 — CHANGES_REQUESTED** [review]
Refer inline comments

**Review by srishtihunjan — CHANGES_REQUESTED** [review]
- remove the duplicate console log
 - remove unnecessary comments
 - remove extra white spaces

**Review by srishtihunjan — APPROVED** [review]

**Review by GITANG1 — APPROVED** [review]

**Pull request merged**